onnxruntime/core/framework/device_stream_collection.h

```diff
@@ -14,7 +14,7 @@ class DeviceStreamCollectionImpl;
 // this collection may be cached and reused for future iterations.
 class DeviceStreamCollection {
  public:
-  DeviceStreamCollection(size_t num_streams, const SessionState& sess_state);
+  DeviceStreamCollection(size_t num_streams, const std::map<OrtDevice, AllocatorPtr>& allocators, bool is_main_graph);
```

Comment:
>std::map<OrtDevice, AllocatorPtr>& allocators [](http://example.com/codeflow?start=51&length=45)

The allocator map seems to be widely used in many places, should we define a type for it?

Reply:
define a new type AllocatorMap instead.